### ISSUE TITLE
DSQL problems

### ISSUE THREAD

**Issue by evertonsales** [nakagami/firebirdsql#31]
Hi,
There are some problems with DSQL statements like:

Insert into TableName (Name, Phone) values ("Everton", "123 123123") returning ID

**Comment by nakagami** [comment]
I think it works.
https://github.com/nakagami/firebirdsql/blob/master/driver_test.go#L133

I want a test case ( or more precise sample code).

**Comment by EvertonSalesBranco** [comment]
I was using a global connection, so, it doesn't execute a "commit" when you use a Query()
So I did something like this:

```
tx, err := globalConn.Begin()
if err != nil {
	panic(err.Error())
}
defer tx.Rollback()
stmt, err := tx.Prepare("INSERT INTO TABLE1 (XXX, YYY, ZZZ) VALUES (?, ?, ?) RETURNING ID")
if err != nil {
	panic(err.Error())
}
defer stmt.Close()
row := stmt.QueryRow(inputJSON.XXX, inputJSON.YYY, inputJSON.ZZZ).Scan(&inputJSON.ID) 
err = tx.Commit()
if err != nil {
	panic(err.Error())
}

```

**Comment by nakagami** [comment]
I hope I have fixed this issue at #35 . Is this issue fixed now ?

**Comment by nakagami** [comment]
Related to issue #76
